docs/pages/guides/infrastructure-management/infrastructure-management.md

```diff
@@ -43,18 +43,28 @@ See [Importing Applications]({{site.baseurl}}guides/configuration-management/imp
 <h4>5. Provisioning</h4>
 <p>A Directed Acyclic Graph (DAG) generated in the previous step is processed. Dependent components are processed sequentially, while others are processed in parallel. Meshery intelligently handles the deployment order to ensure successful deployment.</p>
 </details>
 
 <h3>Auto-Deployment of CRDs and Operators</h3>
+Meshery automates the deployment of Custom Resource Definitions (CRDs) and operators based on the source from which a particular component was registered. When a deployment request is made for a Helm Chart without the upgrade flag, Meshery does not check for the existence of the Helm Chart and always tries to install it. In cases where the chart/release already exists, the installation fails (from the Helm side), and Meshery errors out. Currently, by default, Meshery automatically deploys components that are sourced from Artifact Hub (utilizing Helm Charts).
 
-Meshery automates the deployment of Custom Resource Definitions (CRDs) and operators based on the source from which a particular component was registered. Currently, by default Meshery automatically deploys components that are sourced from Artifact Hub (utilizing Helm Charts). (_Support for OCI registries is expected in the near future._)
-
+Meshery gives users control over installing dependencies through a simple checkbox option. Users can decide whether to install dependencies during deployment. By default, Meshery tries to install dependencies without checking for an existing release, making deployment quicker. (Support for OCI registries is coming soon.)
 {% include alert.html title="Upcoming Features" type="info" content="<p>Meshery is actively developing the ability to import custom models and components, expanding the platform's flexibility. Support for OCI registries is expected in the near future.</p>
 <p>1. <b>Custom Models and Components:</b>
   Meshery is actively developing the ability to import custom models and components, expanding the platform's flexibility.</p>
 <p>2. <b>OCI Registry Support:</b>
   Meshery will soon be adding support for OCI registries in addition to ArtifactHub.
   </p>" %}
+
+<h3>Frequently Asked Questions</h3>
+
+**Q: How does Meshery determine where to get the required dependency for provisioning components?**
+
+**A:** Meshery determines the source based on the metadata of models and components. This metadata includes the actual location of the components, whether it's a Helm chart, Kubernetes manifests, or any other format. Depending on the type of source (Helm chart or Kubernetes manifest), Meshery deploys the components accordingly.
+
+**Q: How does Meshery know what Operator to provision when provisioning components from a given Model?**
```

Comment:
There isn’t a difference on how Meshery determines dependent CRDs and operators.